### DISCUSSION TITLE
Discord Verification and partner

### DISCUSSION THREAD

**Discussion by JustFinchy** [o3de/sig-operations#12] [General]
We need to try and get the Discord Verified to show the Community that it is the official discord. 

Also, once the partner application comes available, apply for that option to unlock Custom banner, Invite splash screen, Unique invite URL and so much more. 

Thought i would also post this so its out there and that everyone can get together and hopefully get the server through the verified process. 
https://discord.com/verification

**Comment by OBWANDO** [comment]
We have emails out with Discord responding back, but they still havent unlocked Partners to continue forward, even though we have met all of the criteria since week of July 25th
